packages/stacked_generator/lib/src/generators/forms/stacked_form_content_generator.dart

```diff
@@ -1,262 +1,41 @@
-import 'package:recase/recase.dart';
-import 'package:stacked_generator/src/generators/base_generator.dart';
-import 'package:stacked_generator/src/generators/forms/field_config.dart';
+import 'package:stacked_generator/src/generators/forms/form_util_service.dart';
 import 'package:stacked_generator/src/generators/forms/form_view_config.dart';
 
 /// A generator that creates the form code based on the configs passed in
-class StackedFormContentGenerator extends BaseGenerator {
-  final FormViewConfig _formViewConfig;
-
-  StackedFormContentGenerator(this._formViewConfig);
+class StackedFormContentGenerator extends FormUtilService {
```

Comment:
I don't like this cause it will expose all the methods of FormUtilService while we want StackedFormContentGenerator to have only one public method which is `generate`